### PR TITLE
fix: Added null propagation and guard

### PR DESCRIPTION
**Pull request by ChapterSevenSeeds**
This PR addresses the issue presented in https://github.com/stephenh/ts-proto/issues/975.

### CODE REVIEW

File: integration/affixes/google/protobuf/timestamp.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
+// Code generated by protoc-gen-ts_proto. DO NOT EDIT.
+// source: google/protobuf/timestamp.proto
+
+/* eslint-disable */
+import { BinaryReader, BinaryWriter } from "@bufbuild/protobuf/wire";
```
Comment (stephenh) [inline integration/affixes/google/protobuf/timestamp.ts:5]: Ah interesting; @ChapterSevenSeeds do you know why these two files were generated now? Was it from your bug fix, or were they just not included on a prior PR? :thinking:

**Review by stephenh — APPROVED** [review]
This is great, thank you!

File: integration/affixes/google/protobuf/timestamp.ts (hunk 1 above)
Comment (ChapterSevenSeeds) [reply integration/affixes/google/protobuf/timestamp.ts:5]: They appeared when I followed the instructions for running the tests. I imagine your assumption that they weren't included from a prior PR is correct 😛

**Pull request merged**